testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -13686,6 +13673,31 @@ void testCastTruncates(CastType castType, SqlOperatorFixture f) {
     }
   }
 
+  static class AliasInfo {
+    private final SqlFunction function;
+    private final String alias;
+    private final SqlLibrary[] libraries;
+
+    private  AliasInfo(SqlFunction function, String alias, SqlLibrary[] libraries) {
```

Comment:
Use a better name than `AliasInfo`. Info means data, after all. Maybe `FunctionAlias`. Also, move it to the end of the file.